mkdocs.yml

```diff
@@ -254,6 +254,9 @@ nav:
           - Monitoring Overview: monitoring-info.md
           - Alerting Overview: alerting-info.md
           - Logging Overview: genestack-logging.md
+      - Accelerated Computing:
```

Comment:
One nit, I think we should put this section under "cloud design."

Reply:
Done